.github/dependabot.yml

```diff
@@ -7,6 +7,17 @@ updates:
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 10
+    ignore:
+      - dependency-name: "com.android.tools.build:gradle"
+        update-types: ["version-update:semver-minor"]
+      - dependency-name: "junit:junit"
+        update-types: ["version-update:semver-minor"]
+      - dependency-name: "org.mockito:*"
+        update-types: ["version-update:semver-minor"]
+      - dependency-name: "androidx.test:*"
+        update-types: ["version-update:semver-minor"]
+      - dependency-name: "org.robolectric:*"
+        update-types: ["version-update:semver-minor"]
```

Comment:
I was going to suggest that we use anchors and aliases to cut down on all of this boilerplate, but [whomp whomp](https://github.com/dependabot/dependabot-core/issues/2127) 🙁